__tests__/workers/postUpdated.ts

```diff
@@ -1198,6 +1262,94 @@ describe('on post update', () => {
     expect(post2).toBeTruthy();
     expect(post2!.scoutId).toEqual('1');
   });
+
+  describe('vordr', () => {
+    it('should update post with the vordr flags on submission with bad scout', async () => {
+      const uuid = randomUUID();
+      const postId = 'vordr3';
+      await saveFixtures(con, Submission, [
+        {
+          id: uuid,
+          url: 'http://vordr.com/test',
+          userId: '1',
+          flags: {
+            vordr: true,
+          },
+        },
+      ]);
+
+      await con.getRepository(ArticlePost).save({
+        id: postId,
+        shortId: postId,
+        url: 'https://post.com/scp1',
+        title: 'Scouted title',
+        visible: false,
+        yggdrasilId: '90660dab-7cd1-49f0-8fe5-41c587ca837f',
+        sourceId: COMMUNITY_PICKS_SOURCE,
+      });
+
+      await expectSuccessfulBackground(worker, {
+        id: '90660dab-7cd1-49f0-8fe5-41c587ca837f',
+        post_id: postId,
+        url: 'http://vordr.com/test',
+        source_id: COMMUNITY_PICKS_SOURCE,
+        submission_id: uuid,
+      });
+
+      const updatedPost = await con.getRepository(ArticlePost).findOneByOrFail({
+        id: postId,
+      });
+
+      expect(updatedPost.flags.vordr).toEqual(true);
+      expect(updatedPost.banned).toEqual(true);
+      expect(updatedPost.flags.banned).toEqual(true);
+      expect(updatedPost.showOnFeed).toEqual(false);
+      expect(updatedPost.flags.showOnFeed).toEqual(false);
+    });
+
+    it('should update post with the vordr flags on submission with good scout', async () => {
+      const uuid = randomUUID();
+      const postId = 'vordr3';
+      await saveFixtures(con, Submission, [
+        {
+          id: uuid,
+          url: 'http://vordr.com/test',
+          userId: '1',
+          flags: {
+            vordr: false,
+          },
+        },
+      ]);
+
+      await con.getRepository(ArticlePost).save({
+        id: postId,
+        shortId: postId,
+        url: 'https://post.com/scp1',
+        title: 'Scouted title',
+        visible: false,
+        yggdrasilId: '90660dab-7cd1-49f0-8fe5-41c587ca837f',
+        sourceId: COMMUNITY_PICKS_SOURCE,
+      });
+
+      await expectSuccessfulBackground(worker, {
+        id: '90660dab-7cd1-49f0-8fe5-41c587ca837f',
+        post_id: postId,
+        url: 'http://vordr.com/test',
+        source_id: COMMUNITY_PICKS_SOURCE,
+        submission_id: uuid,
+      });
+
+      const updatedPost = await con.getRepository(ArticlePost).findOneByOrFail({
+        id: postId,
+      });
+
+      expect(updatedPost.flags.vordr).toEqual(false);
+      expect(updatedPost.banned).toEqual(false);
```

Comment:
Can we add expect to make sure if post had some other flags they did not get overwritten.

Reply:
Added extra test, does that work?